compiler/rustc_passes/src/reachable.rs

```diff
@@ -101,7 +101,7 @@ impl<'tcx> Visitor<'tcx> for ReachableContext<'tcx> {
                     // If this path leads to a constant, then we need to
                     // recurse into the constant to continue finding
                     // items that are reachable.
-                    Res::Def(DefKind::Const | DefKind::AssocConst, _) => {
+                    Res::Def(DefKind::Const | DefKind::AssocConst | DefKind::Static(_), _) => {
```

Comment:
Please revert this change. I assume you're planning on putting up another PR for that reachability bug.

Reply:
Damn, I must have accidentally `git commit -a` at some point 🤦